.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
+      - uses: raboof/release-drafter@v5
```

Comment:
This is cool!

Any reason to use raboof/release-drafter rather than release-drafter/release-drafter?

Reply:
This way if release-drafter is somehow compromised in the future, we won't be immediately affected. Perhaps a bit :potato: :potato: of course, but directly giving externally managed code access to our repos makes me a bit uneasy :smile:

Reply:
OK.  And I trust you not to 💩 our repos 😛